test/system/reports_test.rb

```diff
@@ -620,4 +620,16 @@ class ReportsTest < ApplicationSystemTestCase
 
     assert height < after_height
   end
+
+  test 'display report interval for mentor while undoing wip' do
+    visit_with_auth report_path(reports(:report32)), 'komagata'
+    assert_selector '.a-page-notice.is-only-mentor.is-danger', text: '10日ぶりの日報です'
+
+    visit_with_auth report_path(reports(:report33)), 'kananashi'
+    click_link '内容修正'
+    click_button '提出'
+
+    visit_with_auth report_path(reports(:report32)), 'komagata'
+    assert_selector '.a-page-notice.is-only-mentor', text: '9日ぶりの日報です'
```

Comment:
こちら、赤字になっても検出されないようです。「10日ぶりの行」で`.is-danger`を消してやってみるとPassになるかと思います。これは、`.is-danger`の上位階層の`.a-page-notice.is-only-mentor`としていることで、`.is-danger`の有無に関わらず、`true`となるようです。下記ではいかがでしょうか〜？

```suggestion
    assert_no_selector '.a-page-notice.is-only-mentor.is-danger', text: '9日ぶりの日報です'
```

Reply:
> .is-dangerの上位階層の.a-page-notice.is-only-mentorとしていることで、.is-dangerの有無に関わらず、trueとなるようです。

そちらの点の確認が漏れていました💦改善策までありがとうございます🙏
[5c9d298](https://github.com/fjordllc/bootcamp/pull/4170/commits/5c9d298df497faf7d4d172f5544ecf9e34c1f5d4)で修正しています。